lib/lpc55-hashcrypt/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "lpc55-hashcrypt"
```

Comment:
I know I'm the one who suggested the name but I think we should probably call this something more specific, maybe `lpc55-hashcrypt-irq`